level/chunk.go

```diff
@@ -190,6 +190,9 @@ func ChunkToSave(c *Chunk, dst *save.Chunk) (err error) {
 		s.BlockLight = v.BlockLight
 	}
 	dst.Sections = sections
+	if dst.Heightmaps == nil {
+		dst.Heightmaps = map[string][]uint64{}
```

Comment:
To maintain consistency with other parts of the entire project.
`make(map[string][]uint64{})`